saleor/order/tests/test_fetch_order_prices.py

```diff
@@ -23,6 +23,16 @@ def order_with_lines(order_with_lines):
     return order_with_lines
 
 
+@pytest.fixture
+def channel_USD(channel_USD):
```

Comment:
We already have such fixture, IMO, you shouldn't override existing one, you should create a new one that will be fitting your requirements

Reply:
I want to override it. Please notice I don't use it as a test argument. I need to override it in order to change other fixtures behaviour

Reply:
But it's misleading. I will look for the `channel_USD` fixture in the `tests/fixtures.py` file. It will be confusing, that it was overridden in this file. 

> This way I avoid saving channel settings (extra db calls, time of execution) for each test.
In which place exactly? And still, I'm not sure if it's worth, the solution looks like anty-pattern for me.